examples/website/google-3d/README.md

```diff
@@ -0,0 +1,28 @@
+This is a minimal standalone version of the Google Maps Integration example
+on [deck.gl](http://deck.gl) website.
+
+Note that this example demonstrates using deck.gl with Google Maps. For other base map options, visit the project templates in [get-started](/examples/get-started).
+
+
+### Usage
+
+To run this example, you need a [Google Maps API key](https://developers.google.com/maps/documentation/javascript/get-api-key). You can either set an environment variable:
```

Comment:
Without the Map ID the map will not work as expected, it will revert to the Raster rendering mode. I've updated the README to include the map id env variable